geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommand.java

```diff
@@ -60,12 +60,17 @@ public ResultModel query(
       @CliOption(key = "file", help = "File in which to output the results.",
           optionContext = ConverterHint.FILE) final File outputFile,
       @CliOption(key = CliStrings.QUERY__INTERACTIVE, unspecifiedDefaultValue = "false",
-          help = CliStrings.QUERY__INTERACTIVE__HELP) final boolean interactive) {
-    DataCommandResult dataResult = select(query);
+          help = CliStrings.QUERY__INTERACTIVE__HELP) final boolean interactive,
+      @CliOption(key = CliStrings.MEMBER,
+          optionContext = ConverterHint.MEMBERIDNAME,
+          help = CliStrings.QUERY__MEMBER__HELP) final String memberNameOrId) {
```

Comment:
This new command option should be documented in `query.html.md.erb`. Also, I notice that that page has some inaccuracies that could be fixed while you're updating it. `[--step-name=value]` is listed as an argument, but not represented in the code,  and the `--file` option described in the docs seems to not be used at all in the code. The `--interactive` option is also not used anywhere in the code, or documented, so I wonder if it could just be removed.

Reply:
I can add --member=value to the gfsh query command description.
Not sure about removing --step-name and --file. Is it possible those are present for legacy support (like, they used to be supported in earlier versions, so need to be kept around?)

Reply:
I talked to Jens about it briefly and the `--file` argument is definitely used, it just gets picked up by the interceptor on the gfsh command, not used in the actual command itself. If you try to include the `--step-name` option, then gfsh throws an error: `Option 'step-name' is not available for this command.` The "step-name" string only appears in this one place too, with no other occurrences in the codebase or the docs, so I can't think of a reason to not remove it here.

Reply:
`--step-name` and `--interactive` can be removed in the docs since they are not used by the command anymore, but they can NOT be removed in the option list for backward compatibility reasons (user with old scripts might break).

Reply:
Thanks Dave for helping on the docs.